web/index.jsp

```diff
@@ -29,6 +29,14 @@ include file="projects.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
     cfg.setTitle("Search");
+    String sourceRootPath = cfg.getSourceRootPath();
+    if(sourceRootPath == null || sourceRootPath.isEmpty()){
+    throw new java.io.FileNotFoundException("Configuration File Not Found");
+    }
+    File sourceRootPathFile = RuntimeEnvironment.getInstance().getSourceRootFile();
+    if(sourceRootPathFile.exists() && !sourceRootPathFile.canRead()){
```

Comment:
But when the file does not actually exist then there is no exception which is not correct, is it?